src/main/app/document/CaseDocumentManagementClient.ts

```diff
@@ -0,0 +1,96 @@
+import Axios, { AxiosInstance, AxiosResponse } from 'axios';
+import config from 'config';
+import FormData from 'form-data';
+
+import { getServiceAuthToken } from '../auth/service/get-service-auth-token';
+import type { UserDetails } from '../controller/AppRequest';
+
+export class CaseDocumentManagementClient {
+  client: AxiosInstance;
+
+  CASE_TYPE = 'NFD';
```

Comment:
Is there a reason you didn't want to import CASE_TYPE and JURISDICTION from definition.ts? Do you think it's better to keep them separate?